project_sequence/tests/test_project_sequence.py

```diff
@@ -118,11 +118,15 @@ def test_sequence_unique(self):
     @users("manager")
     def test_project_without_sequence(self):
         """Preexisting projects had no sequence, and they should display fine."""
-        proj1 = self.env["project.project"].create(
-            {"name": "one", "sequence_code": False}
+        proj1 = self.env["project.project"].search(
```

Comment:
I would have added a test too with a create() with "" in sequence_code

Reply:
Thank you @rousseldenis I've just added a new test.